### PR TITLE
feat/footer

### PR DESCRIPTION
**Pull request by shurme**
Task link: https://app.clickup.com/t/86bw5431e
### **Description:** 
Footer for the page.
### **Demo Desktop:**
imagem_2023-11-11_201444847
imagem_2023-11-11_201454106
### **Demo Mobile:**
imagem_2023-11-11_201521197
imagem_2023-11-11_201528495

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -34,45 +35,49 @@ const Footer: FunctionComponent<FooterProps> = ({ lastEditionUrl }) => {
...
-        <a
-          className="linkedin drop-shadow-3xl transition-all duration-300 hover:scale-110 hover:drop-shadow-4xl"
-          href="https://www.linkedin.com/company/nei-isep"
-          target="_blank"
-          rel="noreferrer"
-        >
-          <Linkedin color="white" size={30} />
-        </a>
+      <div className="pt-2">
+        <hr className="lg:w-11/12 md:w-6/12 sm:5/12 w-7/12  h-1 mx-auto my-4 bg-white border-0"></hr>
```
Comment (tomasflopes) [inline src/components/Footer/index.tsx:39]: I like the w-11/12 in all sizes

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -34,45 +35,49 @@ const Footer: FunctionComponent<FooterProps> = ({ lastEditionUrl }) => {
...
       </div>
-
-      <h4 className="select-none py-6 text-center font-good__times text-white drop-shadow-3xl">
-        Copyright &copy; {currentYear} NEI-ISEP. All rights reserved.
-      </h4>
+      <div className="lg:flex justify-between lg:pb-6">
+        <h4 className="select-none sm:text-center lg:pb-0 pb-3 right-10 lg:pl-20 font-good__times text-white drop-shadow-3xl">
```
Comment (tomasflopes) [inline src/components/Footer/index.tsx:42]: font-good__times was removed from the project

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -34,45 +35,49 @@ const Footer: FunctionComponent<FooterProps> = ({ lastEditionUrl }) => {
...
+          Copyright &copy; {currentYear} NEI-ISEP. All rights reserved.
+        </h4>
+        <div className="mb-8 flex justify-center lg:pr-20 items-center gap-5">
+          <a
+            className="facebook drop-shadow-3xl transition-all duration-300 hover:scale-110 hover:drop-shadow-4xl"
```
Comment (tomasflopes) [inline src/components/Footer/index.tsx:47]: facebook, linkedin, instagram & twitter are not a class names

File: src/components/Footer/index.tsx
Comment (tomasflopes) [inline src/components/Footer/index.tsx:1]: Try to improve the NEI logo sizes responsiveness. In small screens it is too large and gets disproportional to the other elements

File: src/components/Footer/index.tsx [hunk 4]
```diff
@@ -16,7 +17,7 @@ const Footer: FunctionComponent<FooterProps> = ({ lastEditionUrl }) => {
       <div className="flex justify-center drop-shadow-lg hover:drop-shadow-xl lg:m-8">
         <div className="my-6 w-max  transition-all duration-300 hover:scale-105">
           <a href={lastEditionUrl} target="_blank" rel="noreferrer">
-            <div className="rounded-3xl border-[5px] border-double border-white p-5 font-good__times text-2xl text-white drop-shadow-3xl transition-all duration-300 lg:text-[2.5rem]">
+            <div className="rounded-2xl bg-secondary p-6 lg:p-7 font-poppins font-bold text-2xl text-white drop-shadow-3xl transition-all duration-300 sm:text-[2.5rem] lg:text-[3.5rem]">
```
Comment (tomasflopes) [inline src/components/Footer/index.tsx:20]: The bg is too dark, check the Figma design

**Review by andrefbarros — APPROVED** [review]

**Pull request merged**